zzre/game/systems/ui/ScrDeck.cs

```diff
@@ -443,10 +443,10 @@ private void CreateGridList(DefaultEcs.Entity entity, ref components.ui.ScrDeck
 
     private IEnumerable<InventoryCard> AllCardsOfType(in components.ui.ScrDeck deck) => deck.ActiveTab switch
     {
-        Tab.Items => deck.Inventory.Items,
-        Tab.Fairies => deck.Inventory.Fairies,
-        Tab.AttackSpells => deck.Inventory.AttackSpells,
-        Tab.SupportSpells => deck.Inventory.SupportSpells,
+        Tab.Items => deck.Inventory.Items.OrderBy(c => c.cardId.EntityId),
```

Comment:
There the sorting is unfortunately not quite correct, but Zanzarah is a bit weird with this:
There is a column in the ItemRow, currently still called `Unknown` (should probably be `Type`) that is used as first sorting term with the order `1 -> 0 -> everything else`
The `EntityId` is used as a secondary sorting term

Reply:
fixed
that's so obscure lol